app/browser/api/ledger.js

```diff
@@ -193,8 +193,13 @@ const paymentPresent = (state, tabId, present) => {
 
 const checkSeed = (state) => {
   const seed = ledgerState.getInfoProp(state, 'passphrase')
+  let localClient = client
 
-  if (seed && !client.isValidPassPhrase(seed)) {
+  if (!localClient) {
+    localClient = require('bat-client')
+  }
+
+  if (seed && typeof localClient === 'object' && !localClient.isValidPassPhrase(seed)) {
```

Comment:
I don't think the check `typeof localClient === 'object'` is needed here. Even if it's null, typeof will return `object`

Reply:
then maybe just plain and simple `localClient`

Reply:
That would be perfect!

Reply:
done